docs/src/components/mdxComponents/Admonition.tsx

```diff
@@ -0,0 +1,110 @@
+import { NextComponentType } from "next";
+import { tm, tmDark, tmHCDark, tmSelectors, media } from "../../themes";
+
+const StyledAdmonition = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  margin: 16px 0;
+  padding: 16px 24px;
+  background-color: ${tm(({ colors }) => colors.tipBackgroundColor)};
+  border-left: 0.5rem solid ${tm(({ colors }) => colors.tipBorderColor)};
+  font-size: 14px;
+  font-family: ChivoRegular;
+  font-weight: 600;
+  line-height: 1.7;
+  letter-spacing: 2px;
+  color: ${tm(({ colors }) => colors.neutral800)};
+  & .title {
+    font-size: 16px;
+  }
+  &.warning {
+    & .title {
+      color: ${tm(({ colors }) => colors.warningColorTitle)};
+    }
+    & p {
+      color: ${tm(({ colors }) => colors.warningColorText)};
+    }
+    border-color: ${tm(({ colors }) => colors.warningBorderColor)};
+    background-color: ${tm(({ colors }) => colors.warningBackgroundColor)};
+  }
+
+  ${tmSelectors.dark} {
+    background-color: ${tmDark(({ colors }) => colors.tipBackgroundColor)};
+    color: ${tmDark(({ colors }) => colors.neutral900)};
+    &.warning {
+      & .title {
+        color: ${tmDark(({ colors }) => colors.warningColorTitle)};
+      }
+      & p {
+        color: ${tmDark(({ colors }) => colors.warningColorText)};
+      }
+      border-color: ${tmDark(({ colors }) => colors.warningBorderColor)};
+      background-color: ${tmDark(
+        ({ colors }) => colors.warningBackgroundColor
+      )};
+    }
+  }
+
+  ${tmSelectors.hcDark} {
+    background-color: ${tmHCDark(({ colors }) => colors.tipBackgroundColor)};
+    color: ${tmHCDark(({ colors }) => colors.neutral900)};
+    &.warning {
+      & .title {
+        color: ${tmHCDark(({ colors }) => colors.warningColorTitle)};
+      }
+      & p {
+        color: ${tmHCDark(({ colors }) => colors.warningColorText)};
+      }
+      border-color: ${tmHCDark(({ colors }) => colors.warningBorderColor)};
+      background-color: ${tmHCDark(
+        ({ colors }) => colors.warningBackgroundColor
+      )};
+    }
+  }
+
+  ${media.mqDark} {
+    ${tmSelectors.auto} {
+      background-color: ${tmDark(({ colors }) => colors.tipBackgroundColor)};
+      color: ${tmDark(({ colors }) => colors.neutral900)};
+      &.warning {
+        & .title {
+          color: ${tmDark(({ colors }) => colors.warningColorTitle)};
+        }
+        & p {
+          color: ${tmDark(({ colors }) => colors.warningColorText)};
+        }
+        border-color: ${tmDark(({ colors }) => colors.warningBorderColor)};
+        background-color: ${tmDark(
+          ({ colors }) => colors.warningBackgroundColor
+        )};
+      }
+    }
+  }
+`;
+
+const Tip: NextComponentType = ({ children }) => {
+  return (
+    <StyledAdmonition>
+      <p className="title">TIP</p>
+      {children}
+    </StyledAdmonition>
+  );
+};
+const Warning: NextComponentType = ({ children }) => {
+  return (
+    <StyledAdmonition className="warning">
```

Comment:
do we need a className here?

Reply:
IMHO - yes.
So we able to not duplicate common CSS properties because of that.